Scripts/posix_test_runner.py

```diff
@@ -40,25 +40,30 @@
             disabled_tests[test] = 1
 
 # run with timeout to avoid locking up
-RunnerArgs = ["timeout", "--signal=9", "300s"]
```

Comment:
All the changes in this file unrelated to gvisor changes?

Reply:
One of the tests in the gvisor suite wouldn't terminate with timeout. There's no reason to use the timeout command since ctest supports this, and can be configured per test or globally